### DISCUSSION TITLE
tart pull does not work as expected

### DISCUSSION THREAD

**Discussion by pavellety** [cirruslabs/tart#195] [Q&A] [answered]
Hi, 

I am trying to pull image from my local registry using this command:

`tart pull --insecure localhost:6000/osx:monterey local-vm-name`

I expect image is pulled from registry and saved as "local-vm-name" so I can see it when type `tart list`. Instead I am getting this error:

```
Error: Unexpected argument 'local-vm-name'
Usage: tart pull <remote-name> [--insecure]
  See 'tart pull --help' for more information.
```

Looks like argument which specify local name is not supported which also indicates help for pull command:

```
tart pull --help
OVERVIEW: Pull a VM from a registry

USAGE: tart pull <remote-name> [--insecure]

ARGUMENTS:
  <remote-name>           remote VM name

OPTIONS:
  --insecure              connect to the OCI registry via insecure HTTP protocol
  --version               Show the version.
  -h, --help              Show help information.
```

My version is 0.20.1

Can you please tell me if this is  bug?

**Comment by edigaryev** [comment, marked as answer]
You can use `tart clone` if you want to pull and clone the image in a single step.

**Reply by pavellety** [reply]
Thanks for a quick reply. Good point. Anyway if possible please fix the documentation README because
 this command `tart pull acme.io/remoteorg/name:latest my-local-vm-name` does not work.

**Reply by edigaryev** [reply]
Thanks for noticing this!

I've created a PR to address this here: https://github.com/cirruslabs/tart/pull/196.
